target/linux/ipq806x/base-files/sbin/g10_check_bootconfig

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+. /lib/functions.sh
+
+mtdnum="$( find_mtd_index 0:BOOTCONFIG )"
+
+if [ ! "$mtdnum" ]; then
```

Comment:
see above.

These g10_* files all have relatively similar content, as well as lib/upgrade/asrock.sh.
I wonder whether it won't be better to put them in a function/library somewhere and add a parameter for switching the few different lines in them?

Reply:
Despite, since those will be available for entire ipq806x, please use asrock_g10 prefix for the file names.